rocket-chat-android-widgets/src/main/java/chat/rocket/android/widget/message/autocomplete/user/UserViewHolder.java

```diff
@@ -73,4 +83,35 @@ private String getImageUrl(String username, AbsoluteUrl absoluteUrl) {
       return null;
     }
   }
+
+  private Drawable getTextDrawable(Context context, String username) {
+    int round = (int) (4 * context.getResources().getDisplayMetrics().density);
+
+    return TextDrawable.builder()
+        .beginConfig()
+        .useFont(Typeface.SANS_SERIF)
+        .endConfig()
+        .buildRoundRect(getUsernameInitials(username), getUserAvatarBackgroundColor(username), round);
+  }
+
+  private String getUsernameInitials(String username) {
```

Comment:
I actually think the widgets should live on the app module... unless they are meant to be reusable, them they should live on another repository...